### ISSUE TITLE
Filtering columns with fonts like wingdings

### ISSUE THREAD

**Issue by kwizimirski** [davidegironi/advanceddatagridview#129]
When I have a column that is using Wingdings or Webdings font to present some symbols, the tree nodes used to build a checklist for filters are made of standard fonts, and I can't see the symbols while I am filtering. I would like to have an option to use the same font family for the nodes as is used in the column that I am trying to filter.
![obraz](https://github.com/davidegironi/advanceddatagridview/assets/146965137/48c881d6-845a-4cfb-9259-85a0026279b4)

**Comment by davidegironi** [comment]
Hello. Can you please build up a testing solution editing the AdvancedDataGridViewSample project?

**Comment by kwizimirski** [comment]
Damn, i dont use github i dont even have any git plugin configured, I use SVN

 

But you can easily add these commands at the end of AddTestData() method

 

 

 

            if (advancedDataGridView_main.Columns.Contains("string"))

            {

                                int columnIndex = advancedDataGridView_main.Columns["string"].Index;

 

                                advancedDataGridView_main.Columns[columnIndex].DefaultCellStyle.Font = new Font("Wingdings", 10);  

            }

 

Then you will see what am i talking about

 

 

 ***@***.***> Krzysztof Wizimirski | kierownik działu programistycznego |  *  ***@***.***> ***@***.***  | +48 609 749 750 

SANTE sp. z o.o. | ul. Jagiellońska 55A | PL 03-301 Warszawa | NIP 524-10-06-610 | GLN: 5900009913804 | BDO:  <https://bdo.mos.gov.pl/web/rejestr-publiczny/podmiot/91736/szczegoly> 000091736 | KRS: 0000950564 | Sąd Rejonowy dla m.st. Warszawy w Warszawie, XIV Wydział Gospodarczy KRS | Kapitał zakładowy: 1.000.000zł |   <http://www.sante.pl/> http://www.sante.pl

Administratorem danych osobowych przetwarzanych w związku z niniejszą korespondencją email jest Sante sp. z o.o. 
Więcej informacji o przetwarzaniu danych osobowych i Twoich prawach dostępne jest w [Polityce prywatności](  <https://sante.pl/polityka-prywatnosci-cookies/> https://sante.pl/polityka-prywatnosci-cookies/)

W trosce o środowisko naturalne prosimy o wydrukowanie tej wiadomości tylko wtedy, gdy jest to konieczne.

**Comment by davidegironi** [comment]
It will suffice. Let me find some time to check it... it will takes a few days to find a slot of free time, please be patient.

**Comment by davidegironi** [comment]
Compile this one: https://file.io/bNE1ooiMlPUK

Use the compiled dll in your project and let me know if it works.
It basically copy the Font style from the DefaultCellStyle.Font property of the selected column name

**Comment by kwizimirski** [comment]
Thank you. Its ALMOST perfect
there is a small bug
when i click the filter for the first time, i have a little image glitch 
![obraz](https://github.com/davidegironi/advanceddatagridview/assets/146965137/86ab3189-ae5e-4ba0-8a63-76fd28d0831f)

then for the second time after i close the filter popup and open it agan, its ok
![obraz](https://github.com/davidegironi/advanceddatagridview/assets/146965137/87ad0a23-d244-44dc-ae33-c2c7ca893af7)

**Comment by kwizimirski** [comment]
This bug only applies to this column, doesnt affect columns with "normal" font
i use exactly this one:
 dataGridViewCellStyle9.Font = new System.Drawing.Font("Wingdings", 12F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(2)));

**Comment by davidegironi** [comment]
try to just set font and dimension to 10 as example.

**Comment by kwizimirski** [comment]
it's not about size I guess, because on 10 there is same issue
![obraz](https://github.com/davidegironi/advanceddatagridview/assets/146965137/5a9eddd1-b30a-4b7c-8c29-d12d9fb99398)

**Comment by davidegironi** [comment]
try add those two line in the "public void Show(Control control, int x, int y, string columnName)" function after "base.Show(control, x, y);", should be almost line 534

```
            //force checklist refresh
            checkList.BeginUpdate();
            checkList.EndUpdate();
```

**Comment by kwizimirski** [comment]
Now its working 100% perfect. thank you

**Comment by davidegironi** [comment]
published as version 1.2.x.12
